documentation/homework/week1/src/App.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
+import './App.css';
+import Header from './components/header';
+import TodoList from './components/todoList';
+import todoList from './components/todoList.json';
+import newID from 'uuid/v4';
+
+
+class App extends Component {
+  state={
+    listTodo:[],
+  }
+  addFunction(id,description,deadline,done) {
+    const currentState = this.state.listTodo;
+    const newState={id:newID(),description:description,deadline:deadline.format('YYYY-MM-DD'),done:done}
+    const updatedState=[...currentState,newState];
+    this.setState({listTodo:updatedState});
```

Comment:
```
this.setState({
listTodo: [
  ...this.state.listTodo,
  {
    id: newID(),
    description,
    deadline: deadline.format('YYYY-MM-DD'),
    done
  }
]
});
```

Don't you think this is neater?, It's just a rewrite for your code but it's up to you how to write it :)